modules/web/src/plugins/try-it/views/http-client.scss

```diff
@@ -225,63 +219,185 @@ hr{
         }
 }
 
-.http-client-select-service-wrapper {
-    padding-left: 15px;
+.http-client-main-wrapper {
     background-color: #333;
-    padding:10px;
+    padding:10px 10px 10px 15px;
     width:auto;
     margin-left: 5px;
-    float: left;
     margin-bottom: 15px;
-    .selectors {
-        float: left;
-        .dropdown-toggle{
-            background-color: #3c3c3c;
-            color: #f1f1f1;
-            text-shadow: none;
-            box-shadow: none;
-            border: 1px solid #575757;
-            width: 200px;
-            margin: 5px 10px;
+    max-width: 900px;
+    .main-wrapper-top {
+        height: 45px;
+        .http-client-request-method-wrapper {
+            float: left;
+            .react-autosuggest__container {
+                .inputContainer {
+                    height: 33px;
+                    padding: 6px;
+                    border: none;
+                    border-bottom: 1px solid #3b3b3b;
+                    input[type=text]{
+                        color: #fff;
+                        background-color: #333;
+                        border: 0px solid #333;
```

Comment:
remove px from 0

Reply:
Fixed in https://github.com/ballerinalang/composer/pull/4354